docs/semantics-of-k.tex

```diff
@@ -6129,8 +6129,10 @@ \subsubsection{Sorts}
 
 <object-sort-constructor> ::= <object-identifier>
 
-<object-sort-list> ::= $\epsilon$ | <object-sort> | <object-sort> `,' 
-<object-sort-list>
+<object-sort-list> ::= $\epsilon$ | <nonempty-object-sort>
```

Comment:
On this line, I think `<nonempty-object-sort>` should be `<nonempty-object-sort-list>`.